docs/releases/2.0.txt

```diff
@@ -207,6 +207,12 @@ Forms
   <django.forms.Form.errors.get_json_data>` method returns form errors as
   a dictionary suitable for including in a JSON response.
 
+* :class:`~django.forms.FileField` as well as :class:`~django.forms.FileInput`
+  and :class:`~django.forms.ClearableFileInput` support a new ``multiple``
+  argument. It allows uploading multiple files using a single file input.
+  In favor of the new argument ``attrs={'multiple': True}`` will emit a
+  warning.
```

Comment:
Last sentence isn't very clear. It contains 2 different ideas and could be made more clear by splitting it into 2 sentences.